src/command_modules/azure-cli-role/azure/cli/command_modules/role/_help.py

```diff
@@ -181,18 +183,49 @@
 """
 helps['ad app permission grant'] = """
     type: command
-    short-summary: Grant an app permissions from another app
+    short-summary: Grant the app an API permission
     examples:
         - name: Grant a native application with permissions to access an existing API with TTL of 2 years
           text: az ad app permission grant --id e042ec79-34cd-498f-9d9f-1234234 --app-id a0322f79-57df-498f-9d9f-12678 --expires 2
 """
 helps['ad app permission list'] = """
     type: command
-    short-summary: List the app permissions
+    short-summary: List API permissions the application has requests
     examples:
         - name: List the OAuth2 permissions for an existing AAD app
           text: az ad app permission list --id e042ec79-34cd-498f-9d9f-1234234
 """
+helps['ad app permission add'] = """
+    type: command
+    short-summary: add an API permission
+    long-summary: invoking "az ad app permission grant" is needed to activate it
+    examples:
+        - name: add a Graph API permission of "Sign in and read user profile"
+          text: az ad app permission add --id eeba0b46-78e5-4a1a-a1aa-cafe6c123456 --api 00000002-0000-0000-c000-000000000000 --api-permissions 311a71cc-e848-46a1-bdf8-97ff7156d8e6=Scope
```

Comment:
Is there a way to identify API permissions by something other than a GUID?  It would be a better UX.

Reply:
I have wished for the similar thing, but unfortunately no. The application's display name is not unique and also, we have 200K+ ones in corporate's tenant